packages/portfolio-contract/test/snapshots/portfolio.contract.test.ts.md

```diff
@@ -93,17 +93,30 @@ Generated by [AVA](https://avajs.dev).
     {
       'orchtest.portfolios.portfolio0': {
         accountIdByChain: {
+          Ethereum: 'eip155:1:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
           agoric: 'cosmos:agoric-3:agoric1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqp7zqht',
           noble: 'cosmos:noble-1:cosmos1test',
         },
         flowCount: 0,
         positionKeys: [
-          'Aave_Base',
+          'Aave_Ethereum',
         ],
       },
-      'orchtest.portfolios.portfolio0.positions.Aave_Base': {
-        accountId: 'eip155:8453:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
+      'orchtest.portfolios.portfolio0.positions.Aave_Ethereum': {
+        accountId: 'eip155:1:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
+        netTransfers: {
+          brand: Object @Alleged: USDC brand {},
+          value: 0n,
+        },
         protocol: 'Aave',
+        totalIn: {
+          brand: Object @Alleged: USDC brand {},
+          value: 0n,
+        },
+        totalOut: {
+          brand: Object @Alleged: USDC brand {},
+          value: 0n,
+        },
```

Comment:
Should these be incremented (and other GMP positions like it, below)?  Refer to USDN tracking (via `trackFlow`?) in other tests.

Reply:
I started, but getting CCTP transfers for GMP working in tests is still TODO:

```
test/portfolio.contract.test.ts
228:// TODO: depositForBurn is throwing
```